PureBasicIDE/ToolbarManagement.pb

```diff
@@ -5,9 +5,9 @@
 ;--------------------------------------------------------------------------------------------
 
 CompilerIf #SpiderBasic
-  #NB_ToolbarMenuItems = 98-24 ; menu items specified in the datasection
+  #NB_ToolbarMenuItems = 99-24 ; menu items specified in the datasection
 CompilerElse
-  #NB_ToolbarMenuItems = 98 ; menu items specified in the datasection
+  #NB_ToolbarMenuItems = 99 ; menu items specified in the datasection
```

Comment:
This is now a conflict with your other merged PR. You should rebase on devel and then change the value to 100 (because both PRs add a new value here).